module-bluetooth/Bluetooth/interface/profiles/HFP/HFP.cpp

```diff
@@ -471,9 +459,10 @@ namespace bluetooth
     }
     void HFP::HFPImpl::initializeCall() const noexcept
     {
-        if (!isAudioRouted) {
-            LOG_DEBUG("Phone number is unused in BT stack - mocking");
-            hfp_ag_outgoing_call_initiated("1234567");
+        if (!isCallInitialized && !isIncomingCall) {
+            LOG_DEBUG("Initializing outgoing call");
+            hfp_ag_outgoing_call_initiated("1234567"); // Mocking phone number - it is not used in stack
```

Comment:
do we have to passed mock phone number here? if so, please create a string called mockNumber or smth like that to show it even more explicitly :smile:

Reply:
Done!